trieste/acquisition/rule.py

```diff
@@ -290,7 +293,75 @@ def acquire(
             )
 
         acquisition_function = self._builder.prepare_acquisition_function(datasets, models)
+
         point = _optimizer.optimize(acquisition_space, acquisition_function)
         state_ = TrustRegion.State(acquisition_space, eps, y_min, is_global)
 
         return point, state_
+
+
+class BatchAcquisitionRule(AcquisitionRule[None, SearchSpace]):
+    """ Implements a acquisition rule for a batch of query points """
+
+    def __init__(self, num_query_points: int,
+                 builder: Optional[AcquisitionFunctionBuilder] = None):
```

Comment:
can you pass any acquisition function builder into this, and it will work? similarly, can you pass a `MonteCarloAcquisition` to `EfficientGlobalOptimization` or `TrustRegion` and expect it to work?

Reply:
having chatted on slack, the answer is no